docs/developer-docs/integrations/sns/integrate-sns/ledger-integration.md

```diff
@@ -1 +1,69 @@
-Title h1: SNS Ledger Integration
\ No newline at end of file
+# Ledger canister
+
+The ledger canister keeps track of the transactions between accounts.
+
+## Initialisation
+
+The index canister initialisation requires the following 
+parameters to be initialized:
+
+```
+type InitArgs = record {
+    minting_account : Account;
+    transfer_fee : nat64;
+    token_symbol : text;
+    token_name : text;
+    metadata : vec record { text; Value };
+    initial_balances : vec record { Account; nat64 };
+    archive_options : record {
+        num_blocks_to_archive : nat64;
+        trigger_threshold : nat64;
+        max_message_size_bytes : opt nat64;
+        cycles_for_archive_creation : opt nat64;
+        node_max_memory_size_bytes : opt nat64;
+        controller_id : principal;
+    };
+};
+```
+
+Example with `dfx`:
```

Comment:
I would again say that this is how it can be initialzed in isolation but developers can also choose to get an SNS that will be deployed with this canister as part of the deployment.
Moreover, all the above initial arguments can be set in the initial arguments of setting up an SNS.

Reply:
Made this point explicit.